### PR TITLE
feat(ci): update docs, optimize multi-arch container build

### PR DESCRIPTION
**Pull request by GoetzGoerisch**

### CODE REVIEW

**Comment by barnstee** [comment]
Hi Goetz! Thanks for your contrib. Did you test with the new NuGets?

**Comment by GoetzGoerisch** [comment]
Hi Erich, yes I did check the basic functionality.

- Connect and subscribe to OPC UA sample server
- Connect and publish to MQTT broker

**Comment by barnstee** [comment]
OK, thanks. I will merge AFTER HMI, but we should also talk about the goals of replacing the Arm64 build instructions. Are you at HMI?

**Comment by barnstee** [comment]
Musste wegen einem Security Issue die NuGets schon updaten, bitte einmal die Aenderungen im Projektfile rausnehmen.

**Comment by GoetzGoerisch** [comment]
@barnstee, could you merge this, or is something needed?

File: README.md [hunk 1]
```diff
@@ -31,34 +31,44 @@ A cross-platform OPC UA cloud publisher reference implementation leveraging OPC
...
+* Support for Store & Forward during internet connection outages
 * Support for username and password authentication
-* Support for both Intel x64 and ARM x64 architectures (Raspberry Pi4, etc.) with pre-built Docker containers ready for use
+* Support for Intel/AMD `x64` and `arm64` architectures (Raspberry Pi4, etc.) with pre-built container images ready for use
 * Integration with [UA Edge Translator](https://github.com/barnstee/UA-EdgeTranslator)
 * Support for generating the Web of Things Thing Descriptions for UA Edge Translator automatically using ChatGPT
 
+## Build Status
+
+[![Docker](https://github.com/umati/UA-CloudPublisher/actions/workflows/docker-publish.yml/badge.svg)](https://github.com/umati/UA-CloudPublisher/actions/workflows/docker-publish.yml)
```
Comment (barnstee) [inline README.md:46]: Please fix link to build status.

**Review by barnstee — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (GoetzGoerisch) [reply README.md]: Thank you for catching this, changed.

**Review by barnstee — APPROVED** [review]

**Pull request merged**